### DISCUSSION TITLE
蛙漫星空链接失效了，求大佬修复下。

### DISCUSSION THREAD

**Discussion by lusking** [xiaohucode/yidaRule#244] [General]
这个链接看漫画很不错，不过从昨天开始好像就网站更换域名了，无法观看了

**Comment by Elvins** [comment]
@xiaohucode 大佬帮忙看看

**Comment by Luckloader** [comment]
同求

**Comment by Hx-yw** [comment]
同求一直都在用漫蛙星空，突然不能用了感觉好难受。

**Comment by QAD21** [comment]
把域名换成：https://manwasp.cc

**Reply by Elvins** [reply]
大佬，规则锁了。自己编辑不了换域名。大佬有密码吗？

**Reply by QAD21** [reply]
没，我重新写了一个，所以重新写一个就能直接解决

**Reply by wuhao1115** [reply]
就是不会写啊，看不懂规则，yds文件不知道怎么做

**Reply by wuhao1115** [reply]
大佬或者给个联系方式呗

**Reply by wuhao1115** [reply]
> 没，我重新写了一个，所以重新写一个就能直接解决

大佬怎么写啊，有啥技巧吗，我试了好多东西，都不能成功，有没有教程啥的

**Reply by QAD21** [reply]
分享给你们：https://share.weiyun.com/JnXh2LM4

**Reply by Sy061003** [reply]
> 分享给你们：https://share.weiyun.com/JnXh2LM4
大佬，想学习一下自己编写这个规则文件，这个文件打开乱码的怎么解决呀？

**Comment by bauw2008** [comment]


**Reply by xbr6** [reply]
大佬复制完剪辑版导入显示格式不正确啊

**Reply by wuhao1115** [reply]
牛的，但是怎么操作啊，我想做个别的网站

**Reply by zhuangzhuangY** [reply]
> 牛的，但是怎么操作啊，我想做个别的网站

您好麻烦您一下，您的满娃可用了吗，能否分享下。

**Reply by QAD21** [reply]
> > 牛的，但是怎么操作啊，我想做个别的网站
> 
> 您好麻烦您一下，您的满娃可用了吗，能否分享下。

自己打开下载就行了，这个是微云的分享：https://share.weiyun.com/JnXh2LM4

**Reply by zhuangzhuangY** [reply]
> > > 牛的，但是怎么操作啊，我想做个别的网站
> > 
> > 
> > 您好麻烦您一下，您的满娃可用了吗，能否分享下。
> 
> 自己打开下载就行了，这个是微云的分享：https://share.weiyun.com/JnXh2LM4

已收到您的文件，十分感谢您

**Reply by QAD21** [reply]
> 
> 已收到您的文件，十分感谢您
不用，小事而已

**Comment by qinghai6749** [comment]
手写了一个漫蛙,txt下载了修改成益达的yds后缀
https://wwtj.lanzouu.com/iEOeq37cy8fa

**Reply by zhuangzhuangY** [reply]
> 手写了一个漫蛙,txt下载了修改成益达的yds后缀 https://wwtj.lanzouu.com/iEOeq37cy8fa

已收到您的文档，十分感谢

**Reply by carryrw** [reply]
大佬 这个只能看60张图片🥹
